WalletWasabi.Gui/Converters/FeeDisplayFormatJsonConverter.cs

```diff
@@ -0,0 +1,55 @@
+using Avalonia.Data.Converters;
+using System;
+using System.Globalization;
+using WalletWasabi.Gui.Models;
+using WalletWasabi.Exceptions;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace WalletWasabi.Gui.Converters
+{
+	public class FeeDisplayFormatJsonConverter : JsonConverter
```

Comment:
It would be nice to add a unit test.

Reply:
Would love to add, there are no current tests for other converters like this. Not familiar enough to go about doing so.

Reply:
Please any question related how to write that test here. (same as above)